pkgs/tools/inputmethods/fcitx5/fcitx5-qt.nix

```diff
@@ -48,6 +48,7 @@ mkDerivation rec {
     homepage = "https://github.com/fcitx/fcitx5-qt";
     license = with licenses; [ lgpl21Plus bsd3 ];
     maintainers = with maintainers; [ poscat ];
+    broken = lib.versionOlder qtbase.version "5.13";
```

Comment:
It is fixable for qt 5.12. Check these patches.
https://github.com/NixOS/nixpkgs/pull/116748/files#diff-ea7a3a933b8f107b11a9c18c604e990761e6c6f590170d5420eb28838c30adc1R25